src/components/MagicCodeInput.js

```diff
@@ -0,0 +1,277 @@
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../styles/styles';
+import * as ValidationUtils from '../libs/ValidationUtils';
+import CONST from '../CONST';
+import TextInput from './TextInput';
+import FormHelpMessage from './FormHelpMessage';
+
+const propTypes = {
+    /** Name attribute for the input */
+    name: PropTypes.string,
+
+    /** Input value */
+    value: PropTypes.string,
+
+    /** Should the input auto focus */
+    autoFocus: PropTypes.bool,
```

Comment:
I used `autoFocus` because it matched the same prop in TextInput, and other components with the same prop also use autoFocus. Should this one be different?

Reply:
yeah, you're right. It's fine to keep the same for now, for the sake of consistency, if there will be a refactor in the other components, will be easy to rename this one as well.